services/user.py

```diff
@@ -0,0 +1,61 @@
+from django.contrib.auth import get_user_model
+
+import settings
+
+
+def create_user(
+    username: str,
+    password: str,
+    email: str = None,
+    first_name: str = None,
+    last_name: str = None
+) -> None:
+
+    new_user = get_user_model().objects.create_user(
+        username=username,
+        password=password
+    )
+
+    if email:
+        new_user.email = email
+
+    if first_name:
+        new_user.first_name = first_name
+
+    if last_name:
+        new_user.last_name = last_name
+
+    new_user.save()
+
+
+def get_user(user_id: int) -> settings.AUTH_USER_MODEL:
+    return get_user_model().objects.get(pk=user_id)
+
+
+def update_user(
+    user_id: int,
+    username: str = None,
+    password: str = None,
+    email: str = None,
+    first_name: str = None,
+    last_name: str = None
```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]